ui/src/backoffice/common/components/Breadcrumbs/__tests__/Breadcrumbs.test.tsx

```diff
@@ -35,4 +35,22 @@ describe('Breadcrumbs', () => {
     const breadcrumbItems = getAllByRole('link');
     expect(breadcrumbItems).toHaveLength(4);
   });
+
+  it('includes the correct link for href2', () => {
+    render(
+      <Provider store={getStore()}>
+        <MemoryRouter initialEntries={[BACKOFFICE]}>
+          <Breadcrumbs
+            title1="Search"
+            href1="backoffice"
+            title2="Author Detail"
+            href2="1234"
+          />
+        </MemoryRouter>
+      </Provider>
+    );
+
+    const link = screen.getByText('Author Detail').closest('a');
+    expect(link).toHaveAttribute('href', expect.stringContaining('/backoffice/1234'));
```

Comment:
what about something like this ? 

```
    const link = screen.getByRole('link', { name: 'Author detail' });
    expect(link).toBeInTheDocument();
    expect(link).toHaveAttribute( ...
```
as getByRole is usually used to find a specific link in a doc for a test